main.go

```diff
@@ -41,102 +41,138 @@ func usage() {
 	yellow.Println("\ttodo a +work +verify did @john fix the build\\?")
 
 	blueBold.Println("\nListing todos")
-	fmt.Println("  When listing todos, you can filter and group the output.\n")
+	fmt.Println("  When listing todos, you can filter and group the output.")
+	fmt.Println()
```

Comment:
Go likes to complain if you have a "\n" on the end of a Println().

Reply:
Yep, good fix!